openmlpimp/utils/search.py

```diff
@@ -20,7 +20,7 @@
 from sklearn.model_selection._split import check_cv
 from sklearn.model_selection._validation import _fit_and_score
 from sklearn.externals.joblib import Parallel, delayed
-from sklearn.utils.fixes import rankdata
```

Comment:
this is deprecated and probably shouldn't be used. Why are you using this? You want to support old versions of scipy? That seems odd...